components/automate-chef-io/content/docs/configuration.md

```diff
@@ -261,32 +263,44 @@ bind_dn = "<your bind_dn>"
 bind_password = "<your bind_password>"
 base_user_search_dn = "<your base user search DN>"
 base_group_search_dn = "<your base group search DN>"
-ca_contents = "<your ca contents>"
-user_query_filter = "<your filter for user queries>"
-group_query_filter = "<your filter for group queries>"
+ca_contents = "<your ca contents>" # optional, but recommended
 ```
 
 The MSAD configuration is an LDAP configuration with more provided default
-values that are commonly a good fit for Active Directory:
+values that are commonly a good fit for Active Directory, but you can override any one by uncommenting it in the config and setting its value:
 
 ```toml
-user_query_filter = "(objectClass=person)" # can be overridden
-group_query_filter = "(objectClass=group)" # can be overridden
-username_attr = "sAMAccountName"
-user_id_attr = "sAMAccountName"
-email_attr = "mail"
-user_display_name_attr = "displayName"
-filter_groups_by_user_attr = "DN"
-filter_groups_by_user_value = "member"
-group_display_name_attr = "displayName"
+[dex.v1.sys.connectors.msad_ldap]
+host = "<your host>"
+bind_dn = "<your bind_dn>"
+bind_password = "<your bind_password>"
+base_user_search_dn = "<your base user search DN>"
+base_group_search_dn = "<your base group search DN>"
+ca_contents = "<your ca contents>" # optional
+
+# MSAD default values (uncomment to override a specific one)
+# insecure_no_ssl = false
```

Comment:
Do these commented-out examples need to have formatting like `"<sAMAccountName>"` with the < > brackets?

Reply:
Not in this case. Those are real values that are the defaults.